kbase-extension/static/kbase/templates/data_staging/ftp_file_table.html

```diff
@@ -17,7 +17,7 @@
             <td>{{mtime}}</td>
             <td>
                 {{#if imported }}
-                <div class="pull-right">
+                <div class="pull-right" style="margin-left: -5px"> <!-- otherwise trashcan goes on new line on resize-->
```

Comment:
No need to add in the comment. @bill / @eamahanna - what is the policy on this kind of inline `style` tag?

Reply:
removed